src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  bookItems: [],
+};
+
+const bookSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: () => {
+      // code implementation to add a book
+    },
+    removeBook: () => {
+      // code implementation to remove a book
+    },
+  },
+});
+export const { addBook, removeBook } = bookSlice.actions;
+export default bookSlice.reducer;
```

Comment:
- [ ] You did well with this booksSlice, but the `addBook` and `removeBook` reducers codes should be implemented. Kindly fix this.

Reply:
I thought that these should be implemented in the next step any way I will fix them.